packages-ui/roosterjs-react/lib/emoji/components/EmojiNavBar.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { css, memoizeFunction } from '@fluentui/react/lib/Utilities';
+import { EmojiFabricIconCharacterMap, EmojiFamilyKeys, EmojiList } from '../utils/emojiList';
+import { FocusZone, FocusZoneDirection } from '@fluentui/react/lib/FocusZone';
+import { Icon } from '@fluentui/react/lib/Icon';
+import { mergeStyleSets } from '@fluentui/react/lib/Styling';
+import { Strings } from '../type/Strings';
+import { Theme, useTheme } from '@fluentui/react/lib/Theme';
+import { TooltipHost } from '@fluentui/react/lib/Tooltip';
+
+/**
+ * @internal
+ * Emoji Nav Bar data
+ */
+export interface EmojiNavBarProps {
+    onClick?: (selected: string) => void;
+    currentSelected?: string;
+    getTabId?: (selected: string) => string;
+    strings: Strings;
```

Comment:
what is this strings for? And you can just use Record<string,string> as type. I don't want to add too many this kind of new types

Reply:
maybe it needs a better name